compile error in k4EDM4hep2LcioConv

Getting this error about a missing edm4hep include file in the latest released version

This should be fixed by some version tag

```
cd /tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-build-wwhcsdb/k4EDM4hep2LcioConv && /opt/spack/lib/spack/env/gcc/g++ -DPODIO_JSON_OUTPUT -Dk4EDM4hep2LcioConv_EXPORTS -I/tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-src/k4EDM4hep2LcioConv/in\
clude -isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/lcio-2.22.1-z7gwfxd3nbozm2qergerlpkkxmzv7s5w/include -isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/edm4hep-0.99-xtvqyvel44oq4ihd2xrxiprvvrcausez/include -isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/podio-1.0.1-qhak3xcmh6m5cidt3om5kywnummvpsk2/include -\
isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/nlohmann-json-3.11.3-q3pgobdq4qbut7o3lgmsw7jgaf3yksnc/include -O3 -DNDEBUG -std=gnu++17 -fPIC -MD -MT k4EDM4hep2LcioConv/CMakeFiles/k4EDM4hep2LcioConv.dir/src/k4Lcio2EDM4hepConv.cpp.o -MF CMakeFiles/k4EDM4hep2LcioConv.dir/src/k4Lcio2EDM4hepConv.cpp.o.d -o CMakeFiles/k4EDM4hep2LcioConv.dir/\
src/k4Lcio2EDM4hepConv.cpp.o -c /tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-src/k4EDM4hep2LcioConv/src/k4Lcio2EDM4hepConv.cpp
cd /tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-build-wwhcsdb/k4EDM4hep2LcioConv && /opt/spack/lib/spack/env/gcc/g++ -DPODIO_JSON_OUTPUT -Dk4EDM4hep2LcioConv_EXPORTS -I/tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-src/k4EDM4hep2LcioConv/in\
clude -isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/lcio-2.22.1-z7gwfxd3nbozm2qergerlpkkxmzv7s5w/include -isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/edm4hep-0.99-xtvqyvel44oq4ihd2xrxiprvvrcausez/include -isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/podio-1.0.1-qhak3xcmh6m5cidt3om5kywnummvpsk2/include -\
isystem /opt/spack/opt/spack/linux-mageia10-zen2/gcc-14.2.0/nlohmann-json-3.11.3-q3pgobdq4qbut7o3lgmsw7jgaf3yksnc/include -O3 -DNDEBUG -std=gnu++17 -fPIC -MD -MT k4EDM4hep2LcioConv/CMakeFiles/k4EDM4hep2LcioConv.dir/src/k4EDM4hep2LcioConv.cpp.o -MF CMakeFiles/k4EDM4hep2LcioConv.dir/src/k4EDM4hep2LcioConv.cpp.o.d -o CMakeFiles/k4EDM4hep2LcioConv.dir/\
src/k4EDM4hep2LcioConv.cpp.o -c /tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-src/k4EDM4hep2LcioConv/src/k4EDM4hep2LcioConv.cpp
In file included from /tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-src/k4EDM4hep2LcioConv/src/k4Lcio2EDM4hepConv.cpp:1:
/tmp/user/spack-stage/spack-stage-k4edm4hep2lcioconv-00-08-02-wwhcsdbmiwigmtbyxy4gpt2xlxtxthli/spack-src/k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4Lcio2EDM4hepConv.h:18:10: fatal error: edm4hep/MCRecoTrackerHitPlaneAssociationCollection.h: No such file or directory
   18 | #include "edm4hep/MCRecoTrackerHitPlaneAssociationCollection.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/packages/k4edm4hep2lcioconv/package.py
+++ b/packages/k4edm4hep2lcioconv/package.py
@@ -19,6 +19,10 @@ class K4edm4hep2lcioconv(CMakePackage, Key4hepPackage):
 
     version("main", branch="main")
     version(
+        "00-09",
+        sha256="bdbb88e2900eb3834d74d100b4d32ae760ee0816ac5fa4a5930772fbe9fb11d9",
+    )
+    version(
         "00-08-02",
         sha256="a0418b5c3c6ce77435bd942279420b0390099f417a7984227cf212710b079321",
     )

